lambdas/matching/src/matching-handler.ts

```diff
@@ -26,12 +26,12 @@ export class MatchingHandler implements LambdaInterface {
       });
       try {
         return {
-          status: response.status,
+          status: response.status.toString(),
```

Comment:
Is this change really necessary?

Reply:
Yes, you cannot pass anything that isn't a string to DynamoDB so it won't save. If you want to save a number e.g. 200, it has to be passed as "200"